### PR TITLE
Added script to rename project stator to the one desired by the user

### LINKED ISSUE #6: Create a script that rename all occurrences of stator and replace it with the user's project name

### PR DESCRIPTION
**Pull request by yann510**
closes #6

### CODE REVIEW

File: renameProject.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const fs = require("fs")
+const path = require("path")
+
+async function* walk(dir, ignoredFolders) {
```
Comment (GabrielBottari) [inline renameProject.js:4]: a star in front of a function keyword?

**Review by GabrielBottari — APPROVED** [review]

**Comment by GabrielBottari** [comment]
Maybe add comment in readme on how to use this

File: renameProject.js (hunk 1 above)
Comment (yann510) [reply renameProject.js:4]: Yes, because it yields

**Review by yann510 — COMMENTED** [review]
Yes, I will add that in the readme for sure.

**Pull request merged**